platform/default/sqlite_cache.cpp

```diff
@@ -210,17 +414,13 @@ void SQLiteCache::Impl::put(const Resource& resource, std::shared_ptr<const Resp
 
 void SQLiteCache::Impl::refresh(const Resource& resource, Seconds expires) {
     try {
-        if (!db) {
-            createDatabase();
-        }
-
-        if (!schema) {
-            createSchema();
-        }
+        initializeDatabase();
 
         if (!refreshStmt) {
-            refreshStmt = std::make_unique<Statement>( //        1               2
-                db->prepare("UPDATE `http_cache` SET `expires` = ? WHERE `url` = ?"));
+            refreshStmt = std::make_unique<Statement>(
+                db->prepare("UPDATE `http_cache` SET `accessed` = strftime('%s', 'now'), "
```

Comment:
This gets converted to INTEGER at the end of the day and AFAIK is the best know method for getting the current Unix epoch inline in the SQL query. But I do agree that it looks odd and probably less efficient than simply binding the value. I will changed that.